node/go.mod

```diff
@@ -304,6 +304,7 @@ replace github.com/wormhole-foundation/wormhole/sdk => ../sdk
 
 replace github.com/wormhole-foundation/wormchain => ../wormchain
 
-replace github.com/CosmWasm/wasmd v0.30.0 => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-1
+// replace github.com/CosmWasm/wasmd v0.30.0 => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-1
+replace github.com/CosmWasm/wasmd => github.com/wormhole-foundation/wasmd v0.30.0-wormchain-1.0.20230706174913-e99aa2ac4302
```

Comment:
TEMP CHANGE until dependent PR https://github.com/wormhole-foundation/wasmd/pull/13 is merged!